node/container.go

```diff
@@ -597,6 +596,31 @@ func (a *HostAgent) setupContainer(tenantID string, svc *service.Service, instan
 	return ctr, state, nil
 }
 
+func (a *HostAgent) getMonitorName(svc *service.Service) string {
```

Comment:
I think a different strategy for determining the name of PID file is necessary.

Control Center makes no assumptions nor places any requirements on the tags associated with a service.  Tags are "implementation details" of the service. This function changes that which, I believe, is contrary to Control Center's design.  Services are written to run in Control Center, but Control Center should not depend on the specific details of the service.

The concept of a "monitor" is specific to Resource Manager and should not be hard-coded into Control Center's implementation.